config-examples/node.yaml

```
@@ -83,6 +83,8 @@ pearl:
   enable_aio: false
   # path to logfile with info about disks states switches, default = "/tmp/bob_events.csv"
   disks_events_logfile: "/tmp/bob_events.csv"
+  # optional, default = false, enables data checksum validation during index regeneration
+  validate_data_checksum_in_index_regen: true
```

You should set its default value here
```suggestion
  validate_data_checksum_in_index_regen: false
```